xtas/core.py

```diff
@@ -84,7 +84,7 @@ def configure(config, import_error="raise", unknown_key="raise"):
 
     app.config_from_object(config.get('CELERY', 'xtas._defaultconfig.CELERY'))
     es = config.get('ELASTICSEARCH', _defaultconfig.ELASTICSEARCH)
-    _config['ELASTICSEARCH'] = es
+    config['ELASTICSEARCH'] = es
```

Comment:
But _config is not private, it's imported by _es.py, so it's a public variable. If that's not intended, then keeping _config and changing _es.py to not use it directly is the way to go.